v2/api/containerregistry/v1api20230701/arm/structure.txt

```diff
@@ -0,0 +1,223 @@
+│   │   ├── DefaultAction: *Enum (2 values)
+│   │   │   ├── "Allow"
+│   │   │   └── "Deny"
+│   │   └── IpRules: Object (2 properties)[]
+│   │       ├── Action: *Enum (1 value)
+│   │       │   └── "Allow"
+│   │       └── Value: *string
+│   ├── Policies: *Object (4 properties)
+│   │   ├── ExportPolicy: *Object (1 property)
+│   │   │   └── Status: *Enum (2 values)
+│   │   │       ├── "disabled"
+│   │   │       └── "enabled"
+│   │   ├── QuarantinePolicy: *Object (1 property)
+│   │   │   └── Status: *Enum (2 values)
+│   │   │       ├── "disabled"
+│   │   │       └── "enabled"
+│   │   ├── RetentionPolicy: *Object (3 properties)
+│   │   │   ├── Days: *int
+│   │   │   ├── LastUpdatedTime: *string
+│   │   │   └── Status: *Enum (2 values)
+│   │   │       ├── "disabled"
+│   │   │       └── "enabled"
+│   │   └── TrustPolicy: *Object (2 properties)
+│   │       ├── Status: *Enum (2 values)
+│   │       │   ├── "disabled"
+│   │       │   └── "enabled"
+│   │       └── Type: *Enum (1 value)
+│   │           └── "Notary"
+│   ├── PrivateEndpointConnections: Object (1 property)[]
+│   │   └── Id: *string
+│   ├── ProvisioningState: *Enum (6 values)
+│   │   ├── "Canceled"
+│   │   ├── "Creating"
+│   │   ├── "Deleting"
+│   │   ├── "Failed"
+│   │   ├── "Succeeded"
+│   │   └── "Updating"
+│   ├── PublicNetworkAccess: *Enum (2 values)
+│   │   ├── "Disabled"
+│   │   └── "Enabled"
+│   ├── Status: *Object (3 properties)
+│   │   ├── DisplayStatus: *string
+│   │   ├── Message: *string
+│   │   └── Timestamp: *string
+│   └── ZoneRedundancy: *Enum (2 values)
+│       ├── "Disabled"
+│       └── "Enabled"
+├── Sku: *Object (2 properties)
+│   ├── Name: *Enum (4 values)
+│   │   ├── "Basic"
+│   │   ├── "Classic"
+│   │   ├── "Premium"
+│   │   └── "Standard"
+│   └── Tier: *Enum (4 values)
+│       ├── "Basic"
+│       ├── "Classic"
+│       ├── "Premium"
+│       └── "Standard"
+├── SystemData: *Object (6 properties)
+│   ├── CreatedAt: *string
+│   ├── CreatedBy: *string
+│   ├── CreatedByType: *Enum (4 values)
+│   │   ├── "Application"
+│   │   ├── "Key"
+│   │   ├── "ManagedIdentity"
+│   │   └── "User"
+│   ├── LastModifiedAt: *string
+│   ├── LastModifiedBy: *string
+│   └── LastModifiedByType: *Enum (4 values)
+│       ├── "Application"
+│       ├── "Key"
+│       ├── "ManagedIdentity"
+│       └── "User"
+├── Tags: map[string]string
+└── Type: *string
+Registry_Spec: Object (6 properties)
+├── Identity: *Object (2 properties)
+│   ├── Type: *Enum (4 values)
+│   │   ├── "None"
+│   │   ├── "SystemAssigned"
+│   │   ├── "SystemAssigned, UserAssigned"
+│   │   └── "UserAssigned"
+│   └── UserAssignedIdentities: map[string]Object (0 properties)
+├── Location: *string
+├── Name: string
+├── Properties: *Object (8 properties)
+│   ├── AdminUserEnabled: *bool
+│   ├── DataEndpointEnabled: *bool
+│   ├── Encryption: *Object (2 properties)
+│   │   ├── KeyVaultProperties: *Object (2 properties)
+│   │   │   ├── Identity: *string
```

Comment:
Looks like this is a clientId field, which should be considered to imported from a ConfigMap.